contribs/gmf/src/controllers/bootstrap.js

```diff
@@ -58,40 +79,51 @@ function bootstrap(module) {
       withCredentials: false,
     },
   });
-  request.fail((jqXHR, textStatus) => {
-    window.alert(`Failed to get the dynamic: ${textStatus}`);
-  });
-  request.done((dynamic) => {
-    if (dynamic.doRedirect) {
-      const small_screen = window.matchMedia ? window.matchMedia('(max-width: 1024px)') : false;
-      if (small_screen && 'ontouchstart' in window) {
-        window.location.href = dynamic.redirectUrl;
+  request
+    .fail((jqXHR, textStatus) => {
+      window.alert(`Failed to get the dynamic: ${textStatus}`);
+    })
+    .done((dynamic) => {
+      if (dynamic.doRedirect) {
+        const small_screen = window.matchMedia ? window.matchMedia('(max-width: 1024px)') : false;
+        if (small_screen && 'ontouchstart' in window) {
+          window.location.href = dynamic.redirectUrl;
+        }
       }
-    }
 
-    if ('gmfCustomJavascriptUrl' in dynamic.constants) {
-      const script = document.createElement('script');
-      script.src = dynamic.constants.gmfCustomJavascriptUrl;
-      script.crossOrigin = 'anonymous';
-      document.body.append(script);
-    }
-    if ('gmfCustomStylesheetUrl' in dynamic.constants) {
-      const style = document.createElement('link');
-      style.href = dynamic.constants.gmfCustomStylesheetUrl;
-      style.rel = 'stylesheet';
-      style.crossOrigin = 'anonymous';
-      document.head.append(style);
-    }
+      if ('gmfCustomJavascriptUrl' in dynamic.constants) {
+        const gmfCustomJavascriptUrl = dynamic.constants.gmfCustomJavascriptUrl;
+        if (Array.isArray(gmfCustomJavascriptUrl)) {
+          gmfCustomJavascriptUrl.forEach((url) => {
+            addScript(url);
+          });
+        } else {
+          addScript(gmfCustomJavascriptUrl);
+        }
+      }
+      if ('gmfCustomStylesheetUrl' in dynamic.constants) {
+        const gmfCustomStylesheetUrl = dynamic.constants.gmfCustomStylesheetUrl;
+        if (Array.isArray(gmfCustomStylesheetUrl)) {
+          gmfCustomStylesheetUrl.forEach((url) => {
+            addStylesheet(url);
+          });
+        } else {
+          addStylesheet(gmfCustomStylesheetUrl);
+        }
+      }
 
-    config.setConfig(dynamic.constants);
+      config.setConfig(dynamic.constants);
 
-    for (const name in dynamic.constants) {
-      module.constant(name, dynamic.constants[name]);
-    }
+      for (const name in dynamic.constants) {
+        module.constant(name, dynamic.constants[name]);
+      }
 
-    setupI18n();
-    angular.bootstrap(document, [`App${interface_}`]);
-  });
+      setupI18n();
+      angular.bootstrap(document, [`App${interface_}`]);
+    })
+    .catch((error) => {
+      window.alert(`Failed to get the dynamic: ${error}`);
```

Comment:
With your `fail` function, we probably don't need this catch anymore. Or at least the message is now wrong

Reply:
Eslint didn't agree, I add it to remove a working that covers all my code...

Reply:
okay...